### DISCUSSION TITLE
Working with an already existing repo

### DISCUSSION THREAD

**Discussion by AYA-123** [astral-sh/rye#611] [Q&A]
![image](https://github.com/mitsuhiko/rye/assets/63795783/eaad2703-3be0-4e27-8f36-6c75743173fe)
I already have .gitignore and a pyproject.toml - so how do I setup and start running rye - I've already installed it.
Sorry for the newbie question!

**Comment by mitsuhiko** [comment]
Running `rye init` should work.

**Comment by AYA-123** [comment]
That gives me the error pyproject.toml already exists. Should I try
deleting it, and then re-run it?

**Reply by DanielAndreasen** [reply]
Did you try `rye sync` standing in the project?
